src/translations/nn.ts

```diff
@@ -14,10 +12,15 @@ const translations = {
     'Uventa feil - Pensjonskalkulator',
   'application.title.beregning': 'Utrekning - Pensjonskalkulator',
   'application.title.forbehold': 'Atterhald - Pensjonskalkulator',
-  'application.title.henvisning_1963':
-    'Tilvisning født før 1963 - Pensjonskalkulator',
-  'application.title.henvisning_ufoere_gjenlevende':
-    'Tilvisning uføretrygd og attlevandepensjon - Pensjonskalkulator',
+  'application.title.henvisning.foedselsdato':
+    'Tilvisning født før 1963 – Pensjonskalkulator',
```

Comment:
Tilvisning -> Tilvising
(ref. [ordbok](https://www.dinordbok.no/norsk-nynorsk/?q=henvisning))

Reply:
Takk. Jeg melder dette til Øyvind for oppfølging.